pkg/cmd/issue/view/issuePrintFormatter.go

```diff
@@ -0,0 +1,132 @@
+package view
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/internal/text"
+	prShared "github.com/cli/cli/v2/pkg/cmd/pr/shared"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/pkg/markdown"
+)
+
+type IssuePrintFormatter struct {
```

Comment:
**Change Requested**

I don't particularly like the unexported functions because it indicates that you can meaningfully call them separately. However, let's table that discussion in light of my other comment about "Tell, Don't Ask".

What I want to call out here is that this type needs a bit of consistency love. The type `IssuePrintFormatter` is exported and the function `NewIssuePrintFormatter` is exported but none of the methods are exported. So I can see the type and I can construct it but I can't do anything with it (except, work with the `IO` field which also happens to be unexported).

If we continue with this pattern, we should make everything unexported.

Reply:
TBH, I've got little rhyme or reason to exported vs unexported functions at this point... Would love to discuss some conventions and hear your opinions about them when we pair on this. I'll then take a pass at making these consistent.